File: exam-slammr-webapp/src/main/java/uk/co/claritysoftware/exam/slammr/webapp/ExamSlammrWebApp.java
package uk.co.claritysoftware.exam.slammr.webapp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.context.config.annotation.RefreshScope;

/**
 * Main Spring Boot bootstrapping class
 */
@SpringBootApplication
@RefreshScope
public class ExamSlammrWebApp {

	public static void main(String[] args) {
		SpringApplication.run(ExamSlammrWebApp.class, args);
	}
}

File: exam-slammr-webapp/src/main/kotlin/uk/co/claritysoftware/exam/slammr/webapp/ExamSlammrWebApp.kt
package uk.co.claritysoftware.exam.slammr.webapp

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.context.config.annotation.RefreshScope

/**
 * Main Spring Boot bootstrapping class
 */
@SpringBootApplication
@RefreshScope
open class ExamSlammrWebApp

fun main(args: Array<String>) {
    SpringApplication.run(ExamSlammrWebApp::class.java, *args)
}